doc/rados/operations/add-or-rm-mons.rst

```diff
@@ -217,43 +217,44 @@ quorum.
 
       ceph-mon -i `hostname` --extract-monmap /tmp/monmap
 
-#. Remove the non-surviving or problematic monitors.  For example, if
-   you have three monitors, ``mon.a``, ``mon.b``, and ``mon.c``, where
-   only ``mon.a`` will survive, follow the example below:
+#. Remove the non-surviving or otherwise problematic monitors:
 
    .. prompt:: bash $
 
       monmaptool {map-path} --rm {mon-id}
 
-   For example,
+   For example, suppose that there are three monitors |---| ``mon.a``, ``mon.b``,
+   and ``mon.c`` |---| and that only ``mon.a`` will survive:
 
    .. prompt:: bash $
 
       monmaptool /tmp/monmap --rm b
       monmaptool /tmp/monmap --rm c
-	
-#. Inject the surviving map with the removed monitors into the
-   surviving monitor(s).  For example, to inject a map into monitor
-   ``mon.a``, follow the example below:
+
+#. Inject the surviving map that includes the removed monitors into the
+   surviving monitor(s):
 
    .. prompt:: bash $
 
       ceph-mon -i {mon-id} --inject-monmap {map-path}
 
-   For example:
+   Continuing with the above example, inject a map into monitor ``mon.a`` by
+   running the following command:
 
    .. prompt:: bash $
 
       ceph-mon -i a --inject-monmap /tmp/monmap
 
+
 #. Start only the surviving monitors.
 
-#. Verify the monitors form a quorum (``ceph -s``).
+#. Verify that the monitors form a quorum by running the command ``ceph -s``.
+
+#. The data directory of the removed monitors is in ``/var/lib/ceph/mon``:
+   either archive this data directory in a safe location or delete this data
+   directory.  However, do not delete it unless you are confident that the
+   remaining monitors are healthy and sufficiently redundant.
```

Comment:
Maybe add a note about ensuring that there is sufficient room for expansion and compaction of the live DB in addition to an archived copy, and possibly compressing the latter.

Reply:
Added.